docs/containers/debug-netcore.md

```diff
@@ -60,3 +80,18 @@ To enable SSL (using the HTTPS protocol), you will need to make a few changes to
    ```
 
 For additional customization options, see the documentation on [Tasks](/docs/containers/reference.md) and [Debug containerized apps](/docs/containers/debug-common.md).
+
+## Saving Project File Preference for [.NET SDK Container Build](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container)
+
+If you have a workspace folder containing multiple .NET project files and you only intend to debug one for the foreseeable future (i.e. you never want to be prompted a list of project files when you `kb(workbench.action.debug.start)`). You can save your launch profile by doing the following:
+
+1. Follow the steps in [.NET SDK Container Build](#net-sdk-container-build-debug-without-dockerfile) and keep the debug session live
+1. Click on the `gear` icon in your debugger view.
+
+   ![dockerSharedFolders](images/debug/debugger-scaffolding.png)
+
+1. Select **Docker: Add Docker Files to Workspace...**
```

Comment:
Wait, can't they scaffold the new debug launch configuration and specify a project file in it, and not have to add a Dockerfile?

Reply:
ahhh shoot... clutch catch! I intended to put `Docker: Debug in Container` and not `Docker: Add Docker Files to Workspace...`